pkg/pxc/app/statefulset/node.go

```diff
@@ -58,7 +58,9 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 	if spec.LivenessInitialDelaySeconds != nil {
 		livenessDelay = *spec.LivenessInitialDelaySeconds
 	}
-
+	if cr.CompareVersionWith("1.6.0") >= 0 {
+		secrets = "internal-" + cr.Name
+	}
```

Comment:
why we fix it here many times? maybe it is better to fix in `pkg/pxc/statefulset.go` once